numba/tests/test_bytes.py

```diff
@@ -0,0 +1,41 @@
+from numba.tests.support import (TestCase, no_pyobj_flags, MemoryLeakMixin)
+from numba import njit
+import unittest
+
+
+def addbytes_len_usecase(a, b):
+    output = len(b''.join([a, b]))
+    return output
+
+
+BYTES_EXAMPLES = [
+    b"abcdef",
+    b"12345",
+    b"AAAA",
+    b"zzzzz",
+    b'z',
+    b'1',
+    bytes(5),
+    bytes(10)
```

Comment:
Given this is used in checking length under concatenation operation(s), there seems to be a lot of duplication of inputs?